src/content/docs/usage/samples.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Samples
+description: Learn how to create and manage samples in Daytona.
+sidebar:
+  label: Samples
+---
+
+import Keyboard from "../../../components/Keyboard.astro"
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
+</Label>
+
+Samples are quick-start repositories with predefined development configurations, environments, and dependencies used to create Workspaces.
+
+## Custom Samples
+
+Daytona provides an option to use custom Samples to create Workspaces. Creating custom Samples is a two-step process that involves:
+
+- [**Sample Repository**](#sample-repository)
+
+    A repository containing the Sample code and a `devcontainer.json` file defining the development environment.
+
+- [**Samples Index**](#samples-index)
+
+    Samples Index (`index.json`) file serving as a registry of available Samples for creating Workspaces.
+
+### Sample Repository
+
+Custom Sample repository is a Git repository that contains the Sample code and a `devcontainer.json` file that defines the development environment. The repository serves as the core content of the Sample and is used to create a Workspace.
+
+1. Create a repository representing the custom Sample.
+
+2. Define the custom Sample code, development configurations and dependencies.
+
+3. Create a **`devcontainer.json`** file that defines the development environment.
+
+- Use the [development container standard](https://containers.dev/) to define your development environment. Development containers vary based on the programming language, configurations, and tools used within the Sample.
+
+Example of a **`devcontainer.json`** file for development environment using Go programming language:
+
+```json
+{
+    "name": "Go",
+    "image": "mcr.microsoft.com/devcontainers/go:1-1.21-bookworm",
+    "customizations": {
+        "vscode": {
+            "extensions": [
+                "golang.Go"
+            ]
+        }
+    },
+    "postCreateCommand": "go test -v"
+}
+```
+
+4. Host the Sample repository publicly on a Git repository or another platform.
```

Comment:
The Sample doesn't have to be public.

Users can host private samples as well. The only requirement is that they have access to that repo when they create a workspace